aiida/manage/configuration/options.py

```diff
@@ -185,6 +185,22 @@
         'description': 'Boolean whether to print AiiDA deprecation warnings',
         'global_only': False,
     },
+    'transport.task_retry_initial_interval': {
+        'key': 'task_retry_initial_interval',
+        'valid_type': 'int',
+        'valid_values': None,
+        'default': 20,
+        'description': 'Initial time interval for the exponential backoff mechanism.',
+        'global_only': False,
+    },
+    'transport.task_maximum_attempts': {
+        'key': 'task_maximum_attempts',
+        'valid_type': 'int',
+        'valid_values': None,
+        'default': 5,
```

Comment:
Something I also noticed is that some defaults are just set in the option `dict`, and some are set by global variables  at the top of the module. We should probably be consistent in this. I'd be in favour of removing the global variables, I think the defaults are easy enough to find.

Reply:
yeh makes sense if they are not used anywhere else